GenderPayGap.WebUI/Models/ScopeNew/OutOfScopeViewModel.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using GovUkDesignSystem;
+using GovUkDesignSystem.Attributes;
+using GovUkDesignSystem.Attributes.ValidationAttributes;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace GenderPayGap.WebUI.Models.ScopeNew
+{
+    [Serializable]
+    public class OutOfScopeViewModel : GovUkViewModel
+    {
+
+        [BindNever]
+        public Database.Organisation Organisation { get; set; }
+        
+        [BindNever]
+        public DateTime ReportingYear { get; set; }
+
+        [GovUkValidateRequired(
+            ErrorMessageIfMissing = "Select a reason why this organisation is out of scope."
+        )]
+        public WhyOutOfScope? WhyOutOfScope { get; set; }
+        
+        [GovUkValidateRequired(
+            ErrorMessageIfMissing = "Please provide a reason that this organisation is out of scope."

```

Comment:
GDS actually recommend against saying "please"(!) - since it makes it sound like it's optional